spk/python312-wheels/Makefile

```diff
@@ -129,8 +139,15 @@ WHEELS_CFLAGS += [msgpack] -std=c11
 endif
 
 # [numpy]
-ifeq ($(call version_gt, $(TC_GCC), 5.0),1)
-WHEELS += src/requirements-crossenv-numpy.txt
+# gcc-8.4 is the minimal version to build numpy
+# gcc-8.5 hapens to be the default version for DSM-7.1
```

Comment:
So, it seems we can no longer build older NumPy versions for DSM versions below 7.1, correct?

Reply:
close enough, we can no longer build older numpy with python 3.12 as no longer compatible.
And newer numpy requires gcc >= 8.4 therefore only supporting DSM-7.1 (as even DSM7 was using gcc7)